packages/d3fc-webgl/src/symbolMapper.js

```
@@ -0,0 +1,25 @@
+import d3Shape from 'd3-shape';
+import circlePointShader from './shaders/point/circle/baseShader';
+import squarePointShader from './shaders/point/square/baseShader';
+
+export default (symbol) => {
+  let outSymbol = d3Shape.symbolCircle;
+  let glSymbol = null;
+
+  switch (symbol) {
+    case d3Shape.symbolCircle:
+      glSymbol = circlePointShader();
+      break;
+    case d3Shape.symbolSquare:
+      glSymbol = squarePointShader();
+      break;
+    default:
```

Please can you make the default throw an unrecognised error rather than default